docs/source/reference/indexing.rst

```diff
@@ -53,6 +53,7 @@ Modifying and computations
    Index.max
    Index.rename
    Index.unique
+   Index.nunique
```

Comment:
Should be good to add it in `MultiIndex` as well.

Reply:
MultiIndex.nunique has been deprecated in `pandas 0.26 API`
https://dev.pandas.io/docs/reference/indexing.html

Reply:
Ah, sure. sounds good!